web/public/templates/common/header.dust

```diff
@@ -41,7 +41,7 @@
           {@i18n key="Community"/}
           <i class="cd-menu__dropdown-icon fa fa-chevron-down"></i>
           <ul>
-            <li><a href="https://coderdojo.com/community/badges/">{@i18n key="Badges"/}</a></li>
+            <li><a href="/badges">{@i18n key="Badges"/}</a></li>
```

Comment:
should this wait for the badges page update and a link be put in the drop down till then?